### PR TITLE
updated packages to latest

### PR DESCRIPTION
**Pull request by RoyHeeffer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,44 +20,46 @@
     "prettier:diff": "prettier --write --config ./.prettierrc --list-different 'src/**/*.ts'"
   },
   "lint-staged": {
-    "*.{ts,tsx}": [
+    "./src/**/*.{ts,tsx}": [
```
Comment (Mrtenz) [inline package.json]: Is this change necessary?

File: src/store.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class TypeormStore extends Store {
   private readonly expirationInterval: number;
   private expirationIntervalId?: number;
 
-  constructor(options: Options) {
+  constructor(options: any) {
```
Comment (Mrtenz) [inline src/store.ts]: This seems like a bad thing. Options for the store should be properly typed.

File: package.json [hunk 3]
```diff
@@ -20,44 +20,46 @@
...
+    "webpack-node-externals": "^3.0.0"
   },
   "husky": {
     "hooks": {
       "post-commit": "git update-index --again",
       "pre-commit": "lint-staged"
     }
+  },
+  "dependencies": {
+    "express-session": "^1.17.3"
```
Comment (Mrtenz) [inline package.json]: Could this be a `peerDependency` instead?

**Review by Mrtenz — COMMENTED** [review]
Thanks! I just have some questions regarding a few changes.

File: package.json (hunk 1 above)
Comment (RoyHeeffer) [reply package.json]: Removed it

File: src/store.ts (hunk 2 above)
Comment (RoyHeeffer) [reply src/store.ts]: Typed it better as it was missing some typing

File: package.json (hunk 3 above)
Comment (RoyHeeffer) [reply package.json]: I followed your example for the typeorm package
